src/plugins/openweathermap/owm/jquery.owm.js

```diff
@@ -0,0 +1,269 @@
+/**
+ * Plugin: jquery.OWM
+ *
+ * Version: 0.0.1
+ * (c) Copyright 2015, Stefan Borchert
```

Comment:
Can you please add licence information (should be the JQuery licence or the GPLv3 in this case, I guess)?